mathjs/mathjs.d.ts

```diff
@@ -1334,6 +1334,10 @@ declare namespace mathjs {
 	export interface MathNode {
                 isNode: boolean;
                 isSymbolNode: boolean;
+                isConstantNode: boolean;
+                isOperatorNode: boolean;
+                op: string;
```

Comment:
I followed the previous convention, also `isSymbolNode` `name` and `value` are only visible on some some node types.
But I agree that it's better to set them optional for clarity.